ERCS/eip-draft-tae-erc20.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Transient Approval Extension for ERC-20
+description: An interface for ERC-20 approvals via EIP-1153 transient storage
+author: Xenia Shape (@byshape), Mikhail Melnik (@ZumZoom)
+discussions-to: # TODO: update
```

Comment:
Please create a topic in https://ethereum-magicians.org/ to discuss this draft ERC.
Include a link in the topic to this PR.

Reply:
Thank you for your comment, all done.